frontend/src/helper/CSVConverter.tsx

```diff
@@ -0,0 +1,37 @@
+import { LogRecord } from "../components/common/types/LogRecord";
+
+const CSVConverter = (data: any): string => {
```

Comment:
Does the LogRecord[] type exist yet? If so, we should be using it here and not any